nodejs.org/s2i/bin/save-artifacts

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+pushd ${HOME} >/dev/null
+
+if [ -d /opt/app-root/src/node_modules ]; then
+  tar -cf /opt/app-root/src/node_modules
```

Comment:
This should be `tar -cf - /opt/app-root/src/node_modules` to send the output to `stdout`.